worlds/LauncherComponents.py

```diff
@@ -116,6 +121,22 @@ def _install_apworld(apworld_src: str = "") -> Optional[Tuple[pathlib.Path, path
     import shutil
     shutil.copyfile(apworld_path, target)
 
+    # If a module with this name is already loaded, then we can't load it now.
+    # TODO: We need to be able to unload a world module,
+    # so the user can update a world without restarting the application.
+    found_already_loaded = False
+    for loaded_world in worlds.world_sources:
+        loaded_name = pathlib.Path(loaded_world.path).stem
+        if module_name == loaded_name:
+            found_already_loaded = True
+            break
+    if found_already_loaded:
+        raise Exception(f"APWorld with the name '{module_name}' is already loaded, "
+                        "so a Launcher restart is required to use the new installation.")
```

Comment:
The message sounds a bit like you need to restart and try again, which would give the same message again. I'd suggest maybe
```suggestion
        raise Exception(f"Installed APWorld successfully, but another with the name '{module_name}' is already loaded,\n"
                        "so a Launcher restart is required to use the new installation.")
```